index.js

```diff
@@ -59,14 +59,54 @@ animalToRemove.remove()
 /************************** EVENTS JS MINI CHALLENGE ******************************/
 
 /***** Deliverable 1 *****/
-function toggleColor(element) {
-    if (element.style.color === "green") {
-        element.style.color = "black"
+
+const toggleHeaderColor = document.querySelector("h1#header")
+
+toggleHeaderColor.addEventListener('click', function (event) {
+    if (toggleHeaderColor.style.color === "green") {
+        toggleHeaderColor.style.color = "black"
     } else {
-        element.style.color = "green"
+        toggleHeaderColor.style.color = "green"
     }
 }
+)
```

Comment:
Nice job! Although we could have used the function as written and used in in the event listener so it is available to reuse with other functions.

We have the header variable already defined earlier in the file.
All we need to do is add an event listener to that header. We provide a callback function to be executed when our "click" event gets triggered.

Great work incorporating the `toggleColor` function. We just need to pass in the event target as an argument to the function.

```
header.addEventListener("click", function(e){
    toggleColor(e.target)
})

```

With ES6 notation

```
header.addEventListener("click", e => toggleColor(e.target))
```